codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -108,8 +108,15 @@ public void channelHandlerContext(ChannelHandlerContext ctx) {
         this.ctx = ctx;
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@Scottmitch maybe just document at the class level that all methods should be called in the event loop? Then get rid of the method-by-method docs?